internal/gcsx/bucket_manager_test.go

```diff
@@ -0,0 +1,127 @@
+package gcsx
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/fsouza/fake-gcs-server/fakestorage"
```

Comment:
i would prefer not getting this dependency out of fake_storage_util.go file. Can you add a method 
StorageHandle([]fakestorage.Object{storage.GetTestFakeStorageObject()}) -> which creates a fakeStorageServer, then storageHandle instance and returns. This way we can avoid making all structs public in storageHandle class and also scope of fakeStorageServer will be limited to fake_storagE_util..


StorageHandle() {
server fakestorage.Server =  CreateFakeStorageServer(GetTestFakeStorageObject())
return &storage.Storageclient{Client: t.fakeStorageServer.Client()}
 
}

Reply:
fixed